src/assets/text/authors.js

```diff
@@ -0,0 +1,82 @@
+export default {
+    // do not delete section. delete individuals as needed. modify content as needed
+    // currently only 'fullName', 'firstName', 'initials', 'profile_link', and 'contribution' are used
+    primaryAuthors: [
+      {
+        firstName: 'Althea1',
+        lastName: 'Archer',
+        fullName: 'Althea Archer1',
+        initials: 'AA1',
+        profile_link: 'https://www.usgs.gov/staff-profiles/althea-archer',
+        role: 'lead developer',
+        contribution: 'led the design and development of the Vue website'
+      },
+      {
+        firstName: 'Althea2',
+        lastName: 'Archer',
+        fullName: 'Althea Archer2',
+        initials: 'AA2',
+        profile_link: 'https://www.usgs.gov/staff-profiles/althea-archer',
+        role: 'lead developer',
+        contribution: 'designed the illustrations'
+      },
+      {
+        firstName: 'Althea3',
+        lastName: 'Archer',
+        fullName: 'Althea Archer3',
+        initials: 'AA3',
+        profile_link: 'https://www.usgs.gov/staff-profiles/althea-archer',
+        role: 'lead developer',
+        contribution: 'made the charts'
+      }
```

Comment:
By this format, if multiple people had the same contribution, they'd get entered as a single author in `authors.js`. Otherwise the entry might be "Althea1 made the charts. Althea2 made the charts. Althea3 made the charts." Instead, you'd use "Althea1, Althea2, and Althea3" and for the contribution "made the charts"

Reply:
Alternatively, you could split by `authors` and `contributions` instead of  `primaryAuthors` and `additionalAuthors`. `authors` could retain names and links, and `contributions` captures roles and contributions. The `role` slot is unused, and instead could define order (i.e. project lead/lead developers first). The `contributions` could capture unique roles instead of unique contributors.

Reply:
Yes, could do. I think you'd need a parameter either in `authors` or `contributions` linking each author to a specific contribution. As it is a pretty different approach, it would mean revamping the current set up. Would you like me to make that change now?

Reply:
With that set up, would we also want one individual to be able to have multiple distinct contributions, e.g. a shared contribution of 'designed the charts', but also a contribution of 'created the illustrations'?

Reply:
> Yes, could do. I think you'd need a parameter either in `authors` or `contributions` linking each author to a specific contribution. As it is a pretty different approach, it would mean revamping the current set up. Would you like me to make that change now?

What would the linking be used to do?

Reply:
Okay - issue added here: #16